### PR TITLE
Part of #17670 Replace Typography with Text component for files: 'create-password.js' & 'creation-successful.js'

### PR DESCRIPTION
**Pull request by PrgrmrHarshShukla**
## Explanation
In this PR I have worked on the issue #17670 Replace Typography with Text component and issue #18714 Replace deprecated design system typography consts with enums for the files:

1.  ui/pages/onboarding-flow/create-password/create-password.js
2.  ui/pages/onboarding-flow/creation-successful/creation-successful.js

### Before

Screenshot (82)

### After
Screenshot (83)

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Hey @PrgrmrHarshShukla, Could you please check for `boxProps` and migrate as well as add screenshots to the PR description thanks

File: ui/pages/onboarding-flow/create-password/create-password.js [hunk 1]
```diff
@@ -190,12 +189,16 @@ export default function CreatePassword({
           marginBottom={4}
         />
       )}
-      <Typography variant={TypographyVariant.H2} fontWeight={FONT_WEIGHT.BOLD}>
+      <Text
+        variant={TextVariant.headingLg}
+        as="h2"
+        fontWeight={FontWeight.Bold}
```
Comment (georgewrmarshall) [inline ui/pages/onboarding-flow/create-password/create-password.js:195]: Shouldn't need this fontWeight as headingLg is already bold
```suggestion
```

**Review by georgewrmarshall — COMMENTED** [review]
Hey @PrgrmrHarshShukla, left one nit and could you please update snapshots then we can get this merged. Thanks for your contribution!

File: ui/pages/onboarding-flow/create-password/__snapshots__/create-password.test.js.snap [hunk 2]
```diff
@@ -30,12 +30,13 @@ exports[`Onboarding Create Password Render should match snapshot 1`] = `
...
     </div>
     <h2
-      class="box box--margin-top-1 box--margin-bottom-1 box--flex-direction-row typography typography--h2 typography--weight-bold typography--style-normal typography--color-text-default"
+      class="box mm-text mm-text--heading-lg box--flex-direction-row box--color-text-default"
     >
       Create password
     </h2>
     <h4
-      class="box box--margin-top-1 box--margin-bottom-1 box--flex-direction-row typography typography--h4 typography--weight-normal typography--style-normal typography--align-center typography--color-text-default"
+      align="center"
```
Comment (georgewrmarshall) [inline ui/pages/onboarding-flow/create-password/__snapshots__/create-password.test.js.snap:38]: Need to update the `align` prop to `textAlign`

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Hey @PrgrmrHarshShukla, The `align` prop is `textAlign` for the Text component could you please update all instances. I noticed it because the `align` prop is being added to the html element in the snapshot

File: ui/pages/onboarding-flow/create-password/__snapshots__/create-password.test.js.snap (hunk 2 above)
Comment (PrgrmrHarshShukla) [reply ui/pages/onboarding-flow/create-password/__snapshots__/create-password.test.js.snap:38]: Will do.

**Comment by PrgrmrHarshShukla** [comment]
Hey @georgewrmarshall ,
Kindly see if this needs to be done.

Screenshot (101)

File: ui/pages/onboarding-flow/creation-successful/creation-successful.js [hunk 3]
```diff
@@ -32,43 +32,45 @@ export default function CreationSuccessful() {
...
-        </Typography>
+        </Text>
       </Box>
-      <Typography
-        variant={TypographyVariant.H4}
-        boxProps={{ align: AlignItems.flexStart }}
+      <Text
+        variant={TextVariant.headingSm}
+        as="h4"
+        align={AlignItems.flexStart}
```
Comment (georgewrmarshall) [inline ui/pages/onboarding-flow/creation-successful/creation-successful.js:52]: I think the boxProps `align` prop was a mistake. It should be `alignItems` and even then I'm not sure why you would use it here

```suggestion
```

**Review by georgewrmarshall — COMMENTED** [review]
Left a comment

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, just wanted to check in. It looks like this PR is almost there. Did you still want to work on it? If so, that's great! Let us know if you need any assistance or if there are any blockers. If you're unable to continue working on it or haven't had a chance to address it, no worries! I can take it from here and carry it forward. Cheers!

**Comment by georgewrmarshall** [comment]
Hey @PrgrmrHarshShukla, thanks for all your effort with this PR I am going to close it as it has become stale. Please feel free to open another PR in future. Thanks for your contributions thus far!

**Pull request closed without merging**